test/e2e/common/runtime.go

```diff
@@ -230,13 +230,18 @@ while true; do sleep 1; done
 			// Images used for ConformanceContainer are not added into NodeImageWhiteList, because this test is
 			// testing image pulling, these images don't need to be prepulled. The ImagePullPolicy
 			// is v1.PullAlways, so it won't be blocked by framework image white list check.
-			imagePullTest := func(image string, hasSecret bool, expectedPhase v1.PodPhase, expectedPullStatus bool) {
+			imagePullTest := func(image string, hasSecret bool, expectedPhase v1.PodPhase, expectedPullStatus bool, windowsImage bool) {
+				command := []string{"/bin/sh", "-c", "while true; do sleep 1; done"}
+				if windowsImage {
```

Comment:
isn't this the same as `framework.NodeOSDistroIs("windows")`, which avoids passing the flag?

Reply:
Given that the specified image needs to be aligned with the command, I like windowsImage as a parameter. I could also imagine an enum, for call site readability, but that would introduce the need to check for other values, so I don't really recommend it.